packages/icons/icons.json

```
@@ -3820,5 +3820,17 @@
         "tags": "symbol, rectangle, filled shape",
         "group": "interface",
         "codepoint": 62242
+    },
+    {
+        "displayName": "Fuel",
+        "iconName": "fuel",
+        "tags": "fuel, gas, gas can, refuel",
+        "group": "interface"
```

I updated the icons contribution guide to mention the new codepoint information required... you'll need to add these:

```suggestion
        "group": "interface",
        "codepoint": 62243
```